examples/app/templates/index.html

```diff
@@ -7,18 +7,18 @@
 <html>
 <head>
   <title>{{page_config['appName'] | e}}</title>
-</head>
-<body>
-    {# Copy so we do not modify the page_config with updates. #}
-    {% set page_config_full = page_config.copy() %}
+  {# Copy so we do not modify the page_config with updates. #}
+  {% set page_config_full = page_config.copy() %}
 
-    {# Set a dummy variable - we just want the side effect of the update. #}
-    {% set _ = page_config_full.update(baseUrl=base_url, wsUrl=ws_url) %}
+  {# Set a dummy variable - we just want the side effect of the update. #}
+  {% set _ = page_config_full.update(baseUrl=base_url, wsUrl=ws_url) %}
 
-    <script id="jupyter-config-data" type="application/json">
-      {{ page_config_full | tojson }}
-    </script>
-    <script src="{{page_config['fullStaticUrl'] | e}}/bundle.js" main="index"></script>
+  <script id="jupyter-config-data" type="application/json">
+    {{ page_config_full | tojson }}
+  </script>
+  <script src="{{page_config['fullStaticUrl'] | e}}/bundle.js" main="index"></script>
```

Comment:
Curious about the reason for moving the `<script>` tag in `<head>`.

Reply:
Maybe for consistency with the main JupyterLab template?

Reply:
Yes this is the reason and also because I added a limit on the page content to be reported in the log (to avoid getting a 100Mo+ log). So I think that config could be of particular interest (especially compared to stylesheets ;) ).

Reply:
:+1: